foxy/distribution.yaml

```diff
@@ -341,6 +341,19 @@ repositories:
       url: https://gitlab.com/autowarefoundation/autoware.auto/autoware_auto_msgs.git
       version: master
     status: developed
+  aws-robomaker-small-warehouse-world:
+    release:
+      packages:
+      - aws_robomaker_small_warehouse_world
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/aws-gbp/aws_robomaker_small_warehouse_world-release.git
+      version: 1.0.0-1
+    source:
+      type: git
+      url: https://github.com/aws-robotics/aws-robomaker-small-warehouse-world.git
```

Comment:
The package.xml says the license is "Apache 2.0", but the license file says "Mit not attribution".
Could you fix that so it's consistent?

Reply:
Thanks for pointing this out. We're going to fix this and then reopen this PR from a different user.